doc/whats_new/v1.6.rst

```diff
@@ -328,6 +328,10 @@ Changelog
   will be returned when `multioutput=uniform_average`.
   :pr:`29709` by :user:`Virgil Chan <virchan>`.
 
+- |API| The default value of the `response_method` parameter of
+  :func:`metrics.make_scorer` will change from `None` to `"predict"` and `None` will be
+  removed in 1.8.
```

Comment:
Maybe we can also mention that in practice it will not change the behaviour of the function.